docs/hardhat-network/README.md

```diff
@@ -1,41 +1,44 @@
-# Hardhat Network
+# Hardhat Network: An Explanation <!--theoretical, not practical; for studying, not for working-->
 
-Hardhat comes built-in with Hardhat Network, a local Ethereum network designed for development. It allows you to deploy your contracts, run your tests and debug your code.
+<!-- TODO: make all RPC method mentions be links to the reference on those methods. -->
+
+Hardhat comes built-in with Hardhat Network, a local Ethereum network node designed for development, akin to Ganache, `geth --devnet`, etc. It allows you to deploy your contracts, run your tests and debug your code.
 
 ## How does it work?
 
-- It mines a block with each transaction that it receives, in order and with no delay.
-- It's backed by the `@ethereumjs/vm` EVM implementation, the same one used by ganache, Remix and Ethereum Studio.
-- It supports the following hardforks:
-  - byzantium
-  - constantinople
-  - petersburg
-  - istanbul
-  - muirGlacier
+It runs as either an in-process or stand-alone daemon, servicing JSON-RPC and WebSocket requests.
+
+It mines a block with each transaction that it receives, in order and with no delay.
```

Comment:
This is now the default, but can be changed, as explained here https://hardhat.org/hardhat-network/#mining-modes

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1657/commits/c1a6d683c159615b60318bfd3842f086c6740ec8